arrows/darknet/darknet_detector.cxx

```diff
@@ -192,7 +191,7 @@ set_configuration( vital::config_block_sptr config_in )
   this->d->m_class_names = config->get_value< std::string >( "class_names" );
   this->d->m_thresh      = config->get_value< float >( "thresh" );
   this->d->m_hier_thresh = config->get_value< float >( "hier_thresh" );
-  this->d->m_gpu_index   = config->get_value< int >( "gpu_index" );
+  this->d->m_gpu_index   = config->get_value< int >( "gpu_index" ); 
```

Comment:
Looks like you've added a space at the end of the line

Reply:
Yeah, I tend to do that to force rebuilds and sometimes forget to undo it